project_mis_report/views/project_project_views.xml

```diff
@@ -29,4 +29,23 @@
         </field>
     </record>
 
+    <record id="project_buttom_kanban_view" model="ir.ui.view">
+        <field name="name">project.project.kanban</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project_kanban" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='name']" position="after">
+                <field name="mis_report_instance_id" invisible="1"/>
+            </xpath>
+            <xpath expr="//span[@class='o_text_overflow']" position="before">

```

Comment:
Does this `xpath` fire an Odoo warning during addon update or installation? Matching by `class` should be discouraged